### ISSUE TITLE
Error initializing a new client

### ISSUE THREAD

**Issue by aredridel** [ssbc/phoenix#355]
Looks like the `name` message gets added before the `init` message.

I get this exception:

```
/Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/secure-scuttlebutt/validation.js:200
      if(!pub && !prev && next.content.type === 'init') {
                                      ^
TypeError: Cannot read property 'type' of undefined
    at validate (/Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/secure-scuttlebutt/validation.js:200:39)
    at /Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/secure-scuttlebutt/validation.js:149:7
    at /Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/secure-scuttlebutt/validation.js:118:16
    at continuation (/Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/cont/node_modules/continuable-para/node_modules/continuable-list/index.js:20:21)
    at /Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/secure-scuttlebutt/index.js:152:24
    at /Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/secure-scuttlebutt/node_modules/level-sublevel/shell.js:101:15
    at /Users/ariastewart/Projects/phoenix/node_modules/scuttlebot/node_modules/secure-scuttlebutt/node_modules/level-sublevel/nut.js:120:19
```
and in the user interface, that the stream has ended.

**Comment by pfraze** [comment]
Thanks for the report, and sorry you're experience this!

So that I can understand the background, can you tell me if your setup process differed from the following?

 1. Install scuttlebot with `npm install -g scuttlebot`
 2. Start scuttlebot with `sbot serve`
 3. Open `localhost:2000` in the browser
 4. Enter a name and click to proceed

If that's correct, I suspect this error occurred after step 4. Is that right?

**Comment by aredridel** [comment]
That's correct.

**Comment by aredridel** [comment]
I installed scuttlebot locally, but ran it with `./node_modules/.bin/sbot server` instead.

**Comment by pfraze** [comment]
Ok! I believe this was a dependency issue -- `scuttlebot` was importing an old version of the web frontend (phoenix). I've published scuttlebot `4.0.1`, which includes the fix. Please give that a shot for me! You should have the `ssbplug-phoenix` dependency at `4.3.2`.

**Comment by aredridel** [comment]
That did it!
